docs/guides/08-animate-your-visualizations.md

```diff
@@ -0,0 +1,410 @@
+const source = new carto.source.Dataset('bird_journey');
+
+// Add better styles
+const viz = new carto.Viz(`
+    @duration: 30
+    @animation: animation($date_time, @duration, fade(0, 0.5))
+    filter: @animation
+    width: 4
+    strokeWidth: 0
+    color: #1785FB
+`);
+
+// Create the layer
+const layer = new carto.Layer('layer', source, viz);
+
+// Get animation control elements
+const $progressRange = document.getElementById('js-progress-range');
+const $playButton = document.getElementById('js-play-button');
+const $pauseButton = document.getElementById('js-pause-button');
+const $durationRange = document.getElementById('js-duration-range');
+const $currentTime = document.getElementById('js-current-time');
+
+// Listen to interaction events
+$playButton.addEventListener('click', () => {
+    viz.variables.animation.play();
+});
+
+$pauseButton.addEventListener('click', () => {
+    viz.variables.animation.pause();
+});
+
+$durationRange.addEventListener('change', () => {
+    viz.variables.duration = parseInt($durationRange.value, 10);
+});
+
+// Update progress each 100 milliseconds
+function updateProgress () {
+    $progressRange.value = viz.variables.animation.getProgressPct();
+    $currentTime.innerText = viz.variables.animation.getProgressValue();
+}
+
+setInterval(updateProgress, 100);
+
+// Add the layer to the map
+layer.addTo(map);
+```
+
+<hr/>
+
+### Taking it further
+In this section, we explore other properties that can be used for your animation, including how to animate only a subset of the data and how to symbolize that time range with color.
+
+In the examples above, we are animating the entire timespan of the bird journeys using `$date_time` as our input. If we want to only visualize a subset of the timespan, we can change the input range of values using a `linear` expression combined with a [`time`](/developers/carto-vl/reference/#cartoexpressionstime) expression:
+
+```js
+const viz = new carto.Viz(`
+    filter: animation(linear($date_time, time('2018-01-01T00:00:00'), time('2018-01-05T00:00:00'))
+`);
+```
+
+**Note:**
+Values outside of the specified range will not appear in the animation.
+
+This isn’t only limited to timestamp properties. If your input property is a number, for example the month of the year ranging from `1-12`, you can adjust the input parameter to that attribute (`$month`) and define the min and max range (`2,6`) that you want to visualize:
+
+```js
+const viz = new carto.Viz(`
+    filter: animation(linear($month, 2, 6))
+`);
+```
+
+You can also apply the same time range (`@time`) to other properties like color using a `ramp` expression. That double-encoding of the same time dimension, within the filter and the color, will reinforce the expressiveness of the visualization:
+
+```js
+const viz = new carto.Viz(`
+    @duration: 10
+    @animation: animation(@time, @duration, fade(0,0.5))
+    @time: linear($date_time, time('2014-03-30T20:24:25Z'), time('2014-04-24T23:52:14Z'))
```

Comment:
I think `@time` is a bad name for this

Reply:
@davidmanzanares any suggestion? how about @timeSteps?